app/config.py

```diff
@@ -426,6 +426,7 @@ class Config(object):
     # feature flag to toggle persistance of notification in celery instead of the API
     FF_NOTIFICATION_CELERY_PERSISTENCE = os.getenv("FF_NOTIFICATION_CELERY_PERSISTENCE", False)
     FF_BATCH_INSERTION = os.getenv("FF_BATCH_INSERTION", False)
+    FF_REDIS_BATCH_SAVING = os.getenv("FF_REDIS_BATCH_SAVING", False)
```

Comment:
is there a manifest PR too?

Reply:
no, not yet! let me go do that...

Reply:
https://github.com/cds-snc/notification-manifests/pull/863